### DISCUSSION TITLE
%cmi_group_maxhomes% for maxhomes of a group of a player.

### DISCUSSION THREAD

**Discussion by CloudeLecaw** [Zrips/CMI#6115] [Enhancement]
its great that we have a placeholder that checks the max placeholders of a player

Let say the player has 10 homes max - then we can check this player has 10 homes max
.
.
CMI allready has the abbility to check the group a player is in.
What i would need is a placeholder that allows to prase the max homes of the group the player is in.

So if Player A is in group Player with max homes of 5. but has additional maxhomes by permission
i can see his max homes but i cant see how much he usualy had by his group.
Also when he changes the group to one with Higher max and got additional once
we cant check how much additional he got without calculating by our own.

So it would be nice to see:
Player has 11 Homes - (default 5 + 2 aditional) (the additionals could be calulated if u can check for the groups
permissions ones)

So i suggest the placeholder:
**%cmi_group_maxhomes%** That shows defined maxhomes of the group the player is in.

**Comment by Zrips** [comment]
There isint any simple way to extract groups permission nodes without adding specific supports for specific permission plugins which increases complexity. Will keep this open as possible addition in future.
